samples/KQL/example_queries/WarehouseManagement.kql

```diff
@@ -0,0 +1,464 @@
+// Telemetry from the SCM Warehouse Management
+
+// Warehouse.LocationDirective.FindLocation

```

Comment:
Should we in the comment for each write which version it is available in. 

Like:
// Warehouse.LocationDirective.FindLocation (10.0.31)

Reply:
It shouldn't hurt and it might be useful.
I'll add it.